raw-data/UW/uw_inventories.tsv

```diff
@@ -53,3 +53,43 @@ roseslabrada2015mako.pdf		Mako		i		i
 						n	"/l/ can also be variably nasalized when followed by nasal vowels"	
 				w		w		
 				ˀw		ˀw		
+
+berry1999abun.pdf	abun1252	Abun		p		p		
+				b		b		
+				t		t		
+				d		d		
+				k		k		
+				ɡ		ɡ		
+				d̠ʒ	"in the Abun Tat dialect to the east there is no separate /j/"	d̠ʒ		
+				mb		mb		
+				nd		nd		
+				n̠d̠ʒ	"Neither does the /nj/ phoneme occur in the Abun Tat dialect."	n̠d̠ʒ		
+				ŋɡ		ŋɡ		
+				f		f		
+				s		s		
+				ʃ		ʃ		
+						t̠ʃ	"A voiceless palatal affricate [tš] occurs in the Abun Ji dialect on occasion in free variation with the palatal fricative [š]"	
+						s	"/š/... neutralises to /s/ syllable-finally"	
+				m		m		
+				n		n		
+				ɲ		ɲ		
+						n̠d̠ʒ	"Neither does the /nj/ phoneme occur in the Abun Tat dialect. Rather it is an allophone of /ñ/"	
+						n	"/ñ/... neutralises to /n/ word-finally"	
+				w		w		
+				r		r		
```

Comment:
The table on page 10 lists this as "glide" which suggests `ɹ` not `r` (a trill).

Reply:
On the bottom of page 13 and top of page 14 it says, "The /r/ is actually a trill, but in fast speech often becomes flap." Should I include ɾ as an allophone?

Reply:
ah, I missed that. Yes, please add the flap as allophone and leave phoneme as `r`.